changes/1222.doc.md

```diff
@@ -0,0 +1 @@
+Small fix in `hikari/api/rest.py` in `fetch_message` docstring. `channel` was mistaken with `message` and the other way around.
```

Comment:
No need for this. We can just skip the fragment check for this pr as it is so small

Reply:
Should I delete the file?

Reply:
Yeah, go ahead :)